lang/user/id.xml

```diff
@@ -344,7 +339,6 @@ Sambungkan Kontroleryang kompatibel dengan SDL2.</not_connected>
 				<cancel_loading>Kamu yakin untuk batalkan loading?</cancel_loading>
 				<no_saved_data>Disini gak ada save data.</no_saved_data>
 				<load_complete>Loading Berhasil.</load_complete>
-				<loading>Loading...</loading>
```

Comment:
Tunggu...

Reply:
is it Malaysia? but this file is for Indonesia.